components/profile-top-menu/dialog.html

```diff
@@ -0,0 +1,16 @@
+<div class="profile-top-menu-floating md-2-line" role="listbox">
+	<md-list-item class="profile-top-menu-item no-pointer" role="option">
```

Comment:
So... why don't use material library mdMenu directive that provides a menu, instead of reimplementing a menu? :m